resources/views/Staff/uploader/index.blade.php

```diff
@@ -0,0 +1,52 @@
+@extends('layout.default')
+
+@section('title')
+    <title>
+        {{ __('torrent.uploader') }} {{ __('common.stats') }} - {{ config('other.title') }}
+    </title>
+@endsection
+
+@section('breadcrumbs')
+    <li class="breadcrumbV2">
+        <a href="{{ route('staff.dashboard.index') }}" class="breadcrumb__link">
+            {{ __('staff.staff-dashboard') }}
+        </a>
+    </li>
+    <li class="breadcrumb--active">{{ __('torrent.uploader') }} {{ __('common.stats') }}</li>
+@endsection
+
+@section('page', 'page__uploader-index')
+
+@section('main')
+    <section class="panelV2">
+        <header class="panel__header">
+            <h2 class="panel__heading">{{ __('torrent.uploader') }} {{ __('common.stats') }}</h2>
+        </header>
+        <div class="data-table-wrapper">
+            <table class="data-table">
+                <thead>
+                    <tr>
+                        <th>{{ __('common.name') }}</th>
+                        <th>{{ __('user.total-uploads') }}</th>
+                        <th>{{ __('user.uploads') }} {{ __('stat.last60days') }}</th>
+                        <th width="15%">{{ __('common.action') }}</th>
```

Comment:
`width` attribute is deprecated. Try css instead.

Reply:
Fixed. Isn't needed. Table looks just fine without it.